src/erlcloud_elb.erl

```diff
@@ -43,17 +44,17 @@
 -define(DESCRIBE_ELBS_PATH, 
         "/DescribeLoadBalancersResponse/DescribeLoadBalancersResult/LoadBalancerDescriptions/member").
 -define(DESCRIBE_ELBS_NEXT_TOKEN, 
-        "/DescribeLoadBalancersResponse/DescribeLoadBalancersResult/NextMarker").
+        "/DescribeLoadBalancersResponse/NextMarker").
```

Comment:
Won't it break anything?

Reply:
I believe that 'NextMarker' should be direct child of '...Response', see e.g. https://github.com/vjache/erlcloud/blob/b2b2601b3aaed992b8939fbd9ebb52fd6750508d/test/erlcloud_route53_tests.erl#L196 or https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/AutoScaling/TDescribeLoadBalancersResponse.html . The last one is a C# model of DescribeLoadBalancersResult and 'Next`marker' property is a direct member of it.